wallet_ui/utils/cycles.tsx

```diff
@@ -0,0 +1,13 @@
+export function format_cycles(cycles : bigint) {
```

Comment:
nit - this file can be a `.ts` file since it doesn't include any React syntax

Reply:
done